### ISSUE TITLE
Cooperate with openusability.org

### ISSUE THREAD

**Issue by jdittrich** [opensourcedesign/organization#67]
OpenUsability (http://openusability.org/) was/is an organization that pushed for a better usability in open source projects, largely by collaborating with students (like Google Summer of Code)

@jdittrich and @Incabell talked to Peter, who was part of the team.

We could ask them if they could link us or if we could even continue their efforts. We currently don't have neither much usability nor cooperation with education, so their efforts might be a great supplement.

**Comment by jancborchardt** [comment]
cc @guiguru (Peter) :) and @bbalazs

**Comment by bbalazs** [comment]
I personally would be very happy if this legal construct with a history could do some good again. It is basically a living dead right now... So do not expect much actual support from here, but I guess we would be more than happy to hand over the brand. It is not my decision though, I am just one member in the Open Usability e.V.

**Comment by jancborchardt** [comment]
@bbalazs yeah, that would be very good to talk about. What were the actual reasons for establishing an e.V., anything in particular? Because for donations, where an e.V. was necessary in the past, we simply use Open Collective: https://opencollective.com/opensourcedesign

Also having been a tiny bit involved in OpenUsability in the past I would welcome merging the brands very much. :) It would make it simpler for people where to get info. We could put the resources and articles or other things from OpenUsability on the opensourcedesign.net website – I assume there are a bunch, sadly the current http://openusability.org website is more of a landing page?
All the members of OpenUsability e.V. who are not added to https://github.com/opensourcedesign yet would also be added. Ideally if you have more contacts, those can join too. :)
Then we could redirect the openusability.org domain to opensourcedesign.net, and state on the Twitter that the resources are now findable there.

@bbalazs @guiguru what do you think? Sorry if it’s a bit too forward, just following on @bbalazs’ proposal above. I always prefer keeping things organizationally simple, and our fields of work overlap sooo much that we should just be one organization.
